src/Components/Onboarding/Components/OnboardingSplitLayout.tsx

```diff
@@ -19,7 +19,7 @@ export const OnboardingSplitLayout: FC<OnboardingSplitLayoutProps> = ({
       <Box
         display={["none", "block"]}
         bg="black100"
-        flexBasis="50%"
+        flexBasis={["30%", "50%"]}
```

Comment:
This doesn't even display at this breakpoint; why would it ever be set to 30%?

Reply:
Whatever it did, did something that worked.. not sure